src/PersistStore.ts

```diff
@@ -142,6 +153,10 @@ export class PersistStore<T, P extends keyof T> {
           actionPersistWarningIf(isActionProperty, propertyName);
 
           if (!isComputedProperty && !isActionProperty) {
+            if (target[propertyName] instanceof ObservableMap) {
+              target[propertyName] = Array.from(target[propertyName].entries());
+            }
+
             propertiesToWatch[propertyName] = toJS(target[propertyName]);
```

Comment:
I think we want this?

```suggestion
            let propertyData = target[propertyName];
            
            if (propertyData instanceof ObservableMap) {
              propertyData = Array.from(propertyData.entries());
            }

            propertiesToWatch[propertyName] = toJS(propertyData);
```

Reply:
it's OK